### PR TITLE
Refactoring of klevr_worker.py with a "inheritance-free" format 

### PR DESCRIPTION
**Pull request by taiya**
Work in progress, gradually migrating features with each commit.
Started form the argparser... now looks like a standard argparse from classical python.

### CODE REVIEW

**Comment by taiya** [comment]
→ moved `max_placement_trials` to the kelvr_worker, as that's something specific to this worker.
→ moved `asset_sources` to kelvr_worker, multi-assets is a atypical scenario, and one that you can deal with yourself in your own worker if necessary (managing bookkeeping in the utils makes it a bit terse to read?)

**Comment by taiya** [comment]
@Qwlouse as promised now the klever_worker can now execute directly inside a Jupyter kernel (inside a container) → see snapshot below :) I still have to do a pass on the portion below line 105, and I would like to move all the remaining function definitions outside of it, so that `klevr_worker.py` will become a purely imperative program.

Screen Shot 2020-10-23 at 11 47 23 PM

**Comment by taiya** [comment]
Hey @Qwlouse, the klever_worker.py is ~done
Only things left to refactor are:
1) add_objects → you were working on that? See that I added a "scene.add" perhaps you can add a multiple dispatch to that method just like we did elsewhere?
2) move_till_no_overlap → will get to this later, as I want to get the TFDS done first

**Comment by taiya** [comment]
@Qwlouse failing tests seem to be orthogonal to my changes?
```
kubric/renderer/blender.py:23: in <module>
    import bpy
E   ModuleNotFoundError: No module named 'bpy'
```

File: kubric/assets/klevr.py [hunk 1]
```diff
@@ -14,36 +14,61 @@
...
-  ]
-
   def __init__(self, uri: str):
     super().__init__(uri)
-    self.spawn_region = ((-3.5, -3.5, 0), (3.5, 3.5, 2))
-    self.ambient_light = (0.05, 0.05, 0.05)
+    #TODO: this seems obsolete?
+    # self.spawn_region = ((-3.5, -3.5, 0), (3.5, 3.5, 2))
+    #TODO: this seems obsolete?
+    # self.ambient_light = (0.05, 0.05, 0.05)
```
Comment (Qwlouse) [inline kubric/assets/klevr.py:45]: ambient_light is desired, but currently not used. Probably got lost somewhere along the way. But the interface will change during my refactoring anyway, so I'll have to bring it back afterwards.

**Review by Qwlouse — COMMENTED** [review]
LGTM. Just some minor comments.

**Comment by Qwlouse** [comment]
> @Qwlouse failing tests seem to be orthogonal to my changes?
> 
> ```
> kubric/renderer/blender.py:23: in <module>
>     import bpy
> E   ModuleNotFoundError: No module named 'bpy'
> ```

Yes, the test-system is currently broken.

**Comment by Qwlouse** [comment]
merged as-is so that I can submit my changes ;-)

**Pull request merged**